packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,410 @@
+<template>
+    <div
+        ref="cookieBanner"
+        :class="[$style['c-cookieBanner'], $style['c-cookieBanner-overlay'], { [$style['c-cookieBanner--hidden']]: hideBanner }]"
+        attr="data-cookie-consent-overlay"
+        :aria-hidden="isHidden">
+        <div
+            v-if="!legacyBanner"
+            :class="$style['c-cookieBanner-card']">
+            <div
+                id="dialog1"
+                :class="$style['c-cookieBanner-content']"
+                role="dialog"
+                aria-labelledby="cookieConsentLabel"
+                aria-modal="true">
+                <h2
+                    id="cookieConsentLabel"
+                    :class="$style['c-cookieBanner-title']">
+                    <a
+                        ref="cookieBannerHeadingFocus"
+                        name="cookieConsentTitle"
+                        tabindex="0"
+                        data-consent-title>
+                        {{ copy.mainTitle }}
+                    </a>
+                </h2>
+
+                <p :class="$style['c-cookieBanner-text']">
+                    {{ copy.text1 }}
+                </p>
+                <p :class="$style['c-cookieBanner-text']">
+                    {{ copy.text2 }}
+                </p>
+                <p
+                    :class="$style['c-cookieBanner-text']">
+                    {{ copy.text3 }}
+                    <a
+                        :href="copy.linkHref"
+                        :class="$style['c-cookieBanner-link']"
+                        target="_blank">
+                        {{ copy.linkText }}</a>{{ copy.text4 }}
+                </p>
```

Comment:
This can be simplified with the `<i18n>` directive. Contact either @FayeCarter or me for more help on this, if you need.

Reply:
Done